web/src/utils/port.ts

```diff
@@ -0,0 +1,11 @@
+import {getService} from 'port-numbers'
```

Comment:
Where is 'port-numbers' coming from?

Reply:
port-numbers is an external package :
https://www.npmjs.com/package/port-numbers